docs/concepts.rst

```diff
@@ -0,0 +1,462 @@
+
+.. _`Concepts`:
+
+Concepts
+========
+
+
+TODO: We need a section on profiles
+
+.. _`The CIM/WBEM Architecture`:
+
+The CIM/WBEM Architecture
+-------------------------
+
+TODO: Write this
+
+.. _`The CIM Model and CIM Objects`:
+
+
+The CIM Model and CIM Objects
+-----------------------------
+
+TODO: Write this
+
+
+.. _`WBEM Operations: Communicating with the WBEM Server`:
+
+WBEM Operations: Communicating with the WBEM Server
+---------------------------------------------------
+
+TODO: Write this section
+
+.. _`Pull Operations: increasing scalability`:
```

Comment:
Incorrect link anchor for "Overview" heading. Note that these anchors are in a global namespace, so having just "Overview" is too ambiguous.